action/main.py

```diff
@@ -38,8 +44,23 @@
 base_cmd = [str(ENV_BIN / "black")]
 if BLACK_ARGS:
     # TODO: remove after a while since this is deprecated in favour of SRC + OPTIONS.
-    proc = run([*base_cmd, *shlex.split(BLACK_ARGS)])
+    proc = run([*base_cmd, *shlex.split(BLACK_ARGS)], stdout=PIPE, stderr=STDOUT)
```

Comment:
```suggestion
    proc = run([*base_cmd, *shlex.split(BLACK_ARGS)], stderr=PIPE, encoding="utf8")
```
All the output you're after here goes to stderr. You can also set the encoding on the call.

Reply:
Regarding what @ichard26 suggested below, if terminal session here is not utf-8 (which is common in windows?), and i'd re-emit the stderr back to console, wouldn't decoding to utf-8 and then re-encoding back to whatever sys.stdout.encoding was cause possible issues ? Not sure about that - asking for clarification